# Stops FluidSynth from forcibly adding linker flags to our build system.
# If we needed them, we'd have added them to meson.build ourselves.
# Also, these ones break Emscripten.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -190,10 +190,10 @@ unset ( ENABLE_UBSAN CACHE )
 
 if ( CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID MATCHES "Clang" OR CMAKE_C_COMPILER_ID STREQUAL "Intel" )
   if ( NOT APPLE AND NOT OS2 )
-    set ( CMAKE_EXE_LINKER_FLAGS
-          "${CMAKE_EXE_LINKER_FLAGS} -Wl,--as-needed" )
-    set ( CMAKE_SHARED_LINKER_FLAGS
-          "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined" )
+
+
+
+
   endif ( NOT APPLE AND NOT OS2 )
 
   # define some warning flags
